src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -54,6 +54,7 @@ import { registerRazorCommands } from './razorCommands';
 import { registerOnAutoInsert } from './onAutoInsert';
 import { commonOptions, languageServerOptions, omnisharpOptions } from '../shared/options';
 import { NamedPipeInformation } from './roslynProtocol';
+import { IDisposable } from '@microsoft/servicehub-framework';
```

Comment:
Should we use the IDisposable we define in this repo?  Probably doesn't matter much

Reply:
Sounds good.